src/dune_rules/pkg_rules.ml

```diff
@@ -601,16 +604,26 @@ module Action_expander = struct
                      (Action.Prog.Not_found.create
                         ~program
                         ~context:t.context
-                        ~loc:(Some loc)
+                        ~loc:None
```

Comment:
Where did the locations go?

Reply:
I removed it because this now takes a `Value.t` which doesn't track the location, but I just changed this function to take the location as an argument obtained from the corresponding slang expression. Also added some tests for this error message to make sure they underline the correct part of the file.